src/main/target/REVO/target.h

```diff
@@ -67,14 +71,33 @@
 #define BEEPER                  NONE
 #endif
 
-// PC0 used as inverter select GPIO
-#define INVERTER_PIN_UART1      PC0
+// UART inverters
+#ifdef AIRBOTF4SD
+#  define INVERTER_PIN_UART6      PD2
```

Comment:
Default in Betaflight seems to be to not indent preprocessor directives. Also, indentation makes grepping for preprocessor directives harder.